pkg/types/identity.go

```diff
@@ -6,8 +6,9 @@ import (
 )
 
 type Identity struct {
-	Key  ed25519.PrivateKey `json:"key"`
-	Nick string             `json:"nick"`
+	Key   ed25519.PrivateKey `json:"key"`
+	Nick  string             `json:"nick"`
+	Admin bool               `json:"admin"`
```

Comment:
Duplicate, maybe just use some kind of struct like:
```
PeerMeta struct {
  Nick string
  Admin bool
  Pic uuid.UUID
}
```
with list or map in room.